src/protocol/data/memcache/request.c

```diff
@@ -39,6 +39,10 @@ request_reset(struct request *req)
     req->delta = 0;
     req->vcas = 0;
 
+    req->partial = false; /* only set to true when partial value received */
```

Comment:
having a `rstate` that may be `REQ_PARTIAL` this sees either redundant, or at least confusing... looking at the change I can see partial being set to `false` only when `rstate` going from `REQ_PARTIAL` to `REQ_PARSED` and set to `true` only when rstate is set to `REQ_PARTIAL`... isn't then this boolean redundant?